test/e2e/service.go

```diff
@@ -966,6 +968,99 @@ var _ = framework.KubeDescribe("Services", func() {
 			framework.Failf("expected un-ready endpoint for Service %v within %v, stdout: %v", t.name, kubeProxyLagTimeout, stdout)
 		}
 	})
+
+	It("should be able to create services of type LoadBalancer and externalTraffic=localOnly [Slow]", func() {
```

Comment:
@lavalamp @ixdy This is going to slow down e2e runs.  We have almost all the load-balancer testing in one case, but this is fundamentally a different test and can't be combined.  Just FYI